We cannot ignore PYTHONPATH here.  Otherwise the test will try to use the
installed multidict.

--- multidict-6.3.2/tests/test_circular_imports.py.orig
+++ multidict-6.3.2/tests/test_circular_imports.py
@@ -81,7 +81,6 @@
     imp_cmd = (
         # fmt: off
         sys.executable,
-        "-I",
         "-W", "error",
         "-c", f"import {import_path!s}",
         # fmt: on
